stores/types.go

```diff
@@ -68,7 +74,7 @@ func (h *hash256) Scan(value interface{}) error {
 	if !ok {
 		return errors.New(fmt.Sprint("failed to unmarshal hash256 value:", value))
 	}
-	if len(bytes) < len(hash256{}) {
+	if len(bytes) != len(hash256{}) {
```

Comment:
`nit` can you change `len(fileContractID{})` to `len(hash256{})` in the line below this one